.github/workflows/tests.yml

```diff
@@ -77,13 +77,27 @@ jobs:
     - name: Scheduled tests
       if:  ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
       run: |
-        pytest tests -m "not integration"
+        if [ ${{ matrix.python-version }} == 3.12 ]; then
+          # TODO(not522): Remove ignores when BoTorch supports Python 3.12
+          pytest tests -m "not integration" \
+            --ignore tests/terminator_tests/ \
+            --ignore tests/visualization_tests/test_terminator_improvement.py
+        else
+          pytest tests -m "not integration"
+        fi
       env:
         SQLALCHEMY_WARN_20: 1
 
     - name: Tests
       if:  ${{ github.event_name != 'schedule' && github.event_name != 'workflow_dispatch' }}
       run: |
-        pytest tests -m "not integration and not slow"
+        if [ ${{ matrix.python-version }} == 3.12 ]; then
+          # TODO(not522): Remove ignores when BoTorch supports Python 3.12
+          pytest tests -m "not integration and not slow" \
+            --ignore tests/terminator_tests/ \
+            --ignore tests/visualization_tests/test_terminator_improvement.py
+        else
+          pytest tests -m "not integration and not slow"
+        fi
```

Comment:
What about this?
I bundled "Tests" and "Scheduled tests", and then the maintenance cost will lower.

```yaml
    - name: Tests
      run: |
        if [ "${{ github.event_name }}" = "schedule" ] || \
           [ "${{ github.event_name }}" = "workflow_dispatch" ] ; then
          target="not integration"
        else
          target="not integration and not slow"
        fi

        if [ "${{ matrix.python-version }}" = "3.12" ] ; then
          # TODO(not522): Remove ignores when BoTorch supports Python 3.12
          ignore_botorch_option="--ignore tests/terminator_tests/ --ignore \
          tests/visualization_tests/test_terminator_improvement.py"
        else
          ignore_botorch_option=""
        fi

        pytest tests -m "$target" $ignore_botorch_option
```